crypto.go

```diff
@@ -75,11 +74,11 @@ func pbDecrypt(info decryptable, password []byte) (decrypted []byte, err error)
 		m := decrypted[:len(decrypted)-psLen]
 		ps := decrypted[len(decrypted)-psLen:]
 		if bytes.Compare(ps, bytes.Repeat([]byte{byte(psLen)}, psLen)) != 0 {
-			return nil, fmt.Errorf("decryption error, incorrect padding")
+			return nil, ErrDecryption // incorrect padding
 		}
 		decrypted = m
 	} else {
-		return nil, fmt.Errorf("decryption error, incorrect padding")
+		return nil, ErrDecryption // incorrect padding
```

Comment:
It looks like all ErrDecryption errors are related to padding, so it could.

This was with regards to "sensitive" information in the thread, though I don't know if I was on the right track or not.